### DISCUSSION TITLE
DirectPV in GitOps

### DISCUSSION THREAD

**Discussion by qjoly** [minio/minio#19912] [Q&A]
Hello,
I have a Kubernetes cluster using DirectPV + Minio Operator to manage several tenants.
I'm having trouble installing + initializing DirectPV: I go through `kubectl directpv discover` and `kubectl directpv init drives.yaml` to find out which disks to use, but is there a way to do this automatically?

My ultimate goal is to be able to have a GitOps workflow from A to Z.

**Comment by balamurugana** [comment]
No. It should not be automated. Init command is destructive and you may lose data for the command run. It is quite rare you run init command often.
